keycloak/keycloak-admin-ui
# Add Save/Cancel buttons and logic to Kerberos settings

## Motivation
https://issues.redhat.com/browse/APPDUX-721

## Brief Description
Adds the ability to save (or cancel) changes made to the User Federation Kerberos settings page.

## Verification Steps
1. Go to User federation and select a Kerberos card.
2. Change any of the values in the Required Settings or Cache Settings sections.
3. Click Save. You should see a success alert.
4. Verify that the changes have been saved by traversing anywhere else in the UI and then re-clicking the card to make sure the values that were changed are displayed correctly.

## Checklist:

## Additional Notes
Screen cap of successful save:
![kerberos-save-success](https://user-images.githubusercontent.com/39063664/104776853-b91ed700-5748-11eb-8f2d-f7f62da96e41.png)
